day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,6 @@
+var i;
+for(i = 10 ; i >= 1 ; i--)
```

Comment:
Nice use of the javascript for loop. A syntax shortcut instead of declaring `var i;` on line 1 is to construct the arguments in the for loop with a `let` statement: `for(let i = 10; i >=1; i--)`